mw-webapp/src/component/tooltip/Tooltip.tsx

```diff
@@ -36,7 +36,10 @@ export const Tooltip = (props: PropsWithChildren<TooltipProps>) => {
       <span className={styles.target}>
         {props.children}
       </span>
-      <div className={classes}>
+      <div
+        className={classes}
+        data-cy="tooltipContent"
```

Comment:
move data-cy to the variable and reuse it in the test

Reply:
need add data-cy prop to tooltip props with optional flag.

Reply:
Agree, and then create variable in the test itself